runtime/jcl/common/mgmtinit.c

```diff
@@ -502,21 +503,32 @@ gcStartEvent(J9JavaVM *vm, UDATA heapSize, UDATA heapUsed, UDATA *totals, UDATA
 
 	mgmt->preCollectionHeapSize = heapSize;
 	mgmt->preCollectionHeapUsed = heapUsed;
+	gcInfo = &gcData->lastGcInfo;
```

Comment:
Should this be done immediate after acquiring the lock so the code above consistently uses the local `gcInfo` pointer rather than sometimes using `gcData->lastGcInfo` still?

I found it confusing to have the two references to the same thing intermixed